server/server.py

```diff
@@ -19,103 +19,48 @@
 #
 
 from accounts import AccountInfo
+from configuration import Configuration
 from datatypes import *
 import gevent.subprocess as sp
 import random
-import socket
 import string
 
-def tuple2ipstring(iptuple):
-    return '%d.%d.%d.%d' % iptuple
+from player_info import PlayerInfo
+from protocols.error import ProtocolViolationError
```

Comment:
I think well have many more errors in the future.

Reply:
Maybe, but I don't think we'll have more modules under protocols. I do see the use in having an import specifically for this exception class, because we'll probably need it in the player states as well.

Can you make a single source file module for this error?